### PR TITLE
Specify ENV Path for pkger for failing CI

### PR DESCRIPTION
**Pull request by firelegacy0**

### CODE REVIEW

**Comment by firelegacy0** [comment]
@MrSaints Will try to use `pkger` again since the current `main.go` uses it for embedding static files, might just be a path issue in the failing CI

File: Dockerfile [hunk 1]
```diff
@@ -45,11 +48,6 @@ LABEL org.label-schema.vcs-url="https://github.com/MrSaints/kubeseal-web" \
 
 RUN apk add --no-cache bash ca-certificates curl jq wget nano
 
-RUN wget https://github.com/bitnami-labs/sealed-secrets/releases/download/v0.17.3/kubeseal-0.17.3-linux-${TARGETARCH}.tar.gz -O kubeseal.tar.gz \
-    && tar -xzf kubeseal.tar.gz -C /tmp/ \
-    && install -m 755 /tmp/kubeseal /usr/local/bin/kubeseal \
-    && rm -rf kubeseal.tar.gz /tmp/kubeseal
```
Comment (MrSaints) [inline Dockerfile]: Why is this removed?

**Comment by MrSaints** [comment]
@firelegacy0 FWIW, Go has [native file embedding](https://pkg.go.dev/embed). How much lift is it to switch to that?

**Comment by firelegacy0** [comment]
> @firelegacy0 FWIW, Go has [native file embedding](https://pkg.go.dev/embed). How much lift is it to switch to that?

Should just require some code change in the libraries/imports used in `main.go` to use the native package, though may require testing to see if it works similar as our current running KubeSeal.

FWIW i think we could:
1) Stick to `pkger` by using this current setup, see if it builds correctly (then it follows the same behavior/setup we have now in KubeSeal)
2) If it doesnt work, will switch to Go's native embed, update `main.go` + `Dockerfile`

File: Dockerfile (hunk 1 above)
Comment (firelegacy0) [reply Dockerfile]: Woops accidentally removed by mistake, oversight from my end
